jsettlers.logic/src/main/java/jsettlers/ai/construction/BestFisherConstructionPositionFinder.java

```diff
@@ -40,13 +40,13 @@ public BestFisherConstructionPositionFinder(EBuildingType buildingType) {
 	@Override
 	public ShortPoint2D findBestConstructionPosition(AiStatistics aiStatistics, AbstractConstructionMarkableMap constructionMap, byte playerId) {
 		List<ScoredConstructionPosition> scoredConstructionPositions = new ArrayList<ScoredConstructionPosition>();
-		int fishDistance = Integer.MAX_VALUE;
```

Comment:
Dont remove the outside-loop variable here but instead set the work area radius here because ... (see next comment)